openapi.yaml

```diff
@@ -3571,6 +3571,87 @@ paths:
       - lang: CLI
         source: >
           linode-cli databases mysql-backups-list 123
+    post:
+      tags:
+      - Databases
+      summary: Managed MySQL Database Backup Snapshot Create
+      operationId: postDatabasesMySQLInstanceBackup
+      servers:
+      - url: https://api.linode.com/v4beta
+      x-linode-cli-action: mysql-backup-snapshot
+      x-linode-grant: unrestricted only
+      description: |
+        Creates a snapshot backup of a Managed MySQL Database.
+
+        Backups generated by this command have the type `snapshot`.
```

Comment:
These snapshots do not show up immediately in the list backups endpoint, should we mention that here? They do show up within a few minutes.